spec/spec_helper.rb

```diff
@@ -0,0 +1,22 @@
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.